### PR TITLE
docs: Add toggle right to left control to playground

### LINKED ISSUE #444: Docs: add RTL toggle for examples
### Description

See fundamental-ngx: https://sap.github.io/fundamental-ngx/action-bar

also see: https://github.com/SAP/fundamental-ngx/pull/511

### Versions
**fundamental-react:** current
**fiori-fundamentals:**

---

_**NOTE:** Where applicable, please include uncropped screen captures._

_**DISCLAIMER:**
After triaging an issue, the fundamental-react team will see if it can be reproduced or confirmed. If more information is needed, the fundamental-react team will contact the author.  Any issues awaiting responses from the author for more than 7 days will be closed.  The author can re-open the issue at a later time if they can present the requested information._

### PR DESCRIPTION
**Pull request by skvale**
### Description
screen shot 2019-02-27 at 5 28 01 am

screen shot 2019-02-27 at 5 28 07 am

fixes #444

### CODE REVIEW

File: src/_playground/documentation/DocsTile/DocsTile.js [hunk 1]
```diff
@@ -2,51 +2,62 @@ import { Button } from '../../../';
...
     constructor(props) {
         super(props);
         this.state = {
-            backgroundToggle: false
+            backgroundToggle: false,
+            rightToLeft: false
         };
     }
 
     handleToggle = () => {
```
Comment (bcullman) [inline src/_playground/documentation/DocsTile/DocsTile.js]: should probably be renamed.

Comment (skvale) [reply src/_playground/documentation/DocsTile/DocsTile.js]: good call e0cb5c4

File: src/_playground/documentation/DocsTile/DocsTile.js [hunk 2]
```diff
@@ -2,51 +2,62 @@ import { Button } from '../../../';
...
+                        size='xs'>
+                        Toggle right to left
+                    </Toggle>
+                    <Toggle
+                        className='frDocs-tile__feature'
+                        inputProps={{ 'aria-label': 'Toggle background color' }}
+                        onChange={this.handleBackgroundToggle}
+                        size='xs'>
+                        Toggle background
+                    </Toggle>
```
Comment (greg-a-smith) [inline src/_playground/documentation/DocsTile/DocsTile.js]: This is probably unfair to put this on you, but we should adjust the verbiage on both of these toggles.  A toggle is an on/off switch so the label should reflect that.  The label "Toggle ..." doesn't really state what the "on" or the "off" state will do.  In fact, the background one is actually probably backwards.  The "off" state shows a background and "on" does not.  I think the default state of both toggles should be "off".  That said, the default for "right to left" is LTR and the default for "background" is to have a background color.

How about these for the labels:
* Show RTL
* Hide background

That way, both can be "off" by default and then it's clear _what_ should happen when the toggle is "on".

**NOTE:** The `aria-label` values should be updated to match.

File: src/_playground/documentation/DocsTile/DocsTile.js [hunk 3]
```diff
@@ -2,51 +2,62 @@ import { Button } from '../../../';
...
-import React, { Component } from 'react';
 
-export class DocsTile extends Component {
+export class DocsTile extends React.Component {
     constructor(props) {
         super(props);
         this.state = {
-            backgroundToggle: false
+            backgroundOn: true,
+            rightToLeft: false
```
Comment (greg-a-smith) [inline src/_playground/documentation/DocsTile/DocsTile.js]: Per comment below, maybe change these state variables to `hideBackground` and `showRTL` so both can be defaulted to `false` (matching the visual "off" state of the toggle).

File: src/_playground/documentation/DocsTile/DocsTile.js (hunk 2 above)
Comment (skvale) [reply src/_playground/documentation/DocsTile/DocsTile.js]: I agree.  
I think the label `Show right to left` would be an improvement to `Show RTL` though

File: src/_playground/documentation/DocsTile/DocsTile.js [hunk 4]
```diff
@@ -2,51 +2,62 @@ import { Button } from '../../../';
...
-                    size='xs'>Toggle background</Toggle>
-                <div className={innerDivClasses}>
-                    {children}
+                <div className='frDocs-tile__features'>
+                    <Toggle
+                        className='frDocs-tile__feature'
+                        inputProps={{ 'aria-label': 'Show right to Left' }}
+                        onChange={this.toggleRTL}
+                        size='xs'>
+                        Show right to Left
```
Comment (jbadan) [inline src/_playground/documentation/DocsTile/DocsTile.js]: Can we fix the capitalization on "Left" then I think this is good to merge!

File: src/_playground/documentation/DocsTile/DocsTile.js [hunk 5]
```diff
@@ -2,51 +2,62 @@ import { Button } from '../../../';
...
-                    className='frDocs-tile__background-toggle'
-                    inputProps={{ 'aria-label': 'Toggle background color' }}
-                    onChange={this.handleToggle}
-                    size='xs'>Toggle background</Toggle>
-                <div className={innerDivClasses}>
-                    {children}
+                <div className='frDocs-tile__features'>
+                    <Toggle
+                        className='frDocs-tile__feature'
+                        inputProps={{ 'aria-label': 'Show right to Left' }}
```
Comment (greg-a-smith) [inline src/_playground/documentation/DocsTile/DocsTile.js]: One minor nit and then I'm on board.  The "L" in "Left" should be lowercase.

File: src/_playground/documentation/DocsTile/DocsTile.js (hunk 4 above)
Comment (greg-a-smith) [inline src/_playground/documentation/DocsTile/DocsTile.js]: One minor nit and then I'm on board.  The "L" in "Left" should be lowercase.

**Comment by greg-a-smith** [comment]
@skvale I changed this to be prefixed with `docs` rather than `feat`.  Although it is a feature for the documentation pages, nothing functional changed for the distribution package so it doesn't seem worthy of a minor version bump.

**Pull request merged**